pkg/planner/cardinality/selectivity.go

```diff
@@ -417,6 +434,39 @@ OUTER:
 	return ret, nodes, nil
 }
 
+// CalcTotalSelectivityForMVIdxPath calculates the total selectivity for the given partial paths of an MV index merge path.
+// It corresponds with the meaning of AccessPath.CountAfterAccess, as used in buildPartialPathUp4MVIndex.
+// It uses the independence assumption to estimate the selectivity.
+func CalcTotalSelectivityForMVIdxPath(
+	coll *statistics.HistColl,
+	partialPaths []*planutil.AccessPath,
+	isIntersection bool,
+) float64 {
+	selectivities := make([]float64, 0, len(partialPaths))
+	for _, path := range partialPaths {
+		// Note that although the total row count of the mv index is usually larger than the table, the CountAfterAccess
+		// of a partial path will never be larger than the table total row count.
+		// And for an index merge path with only one partial path, the CountAfterAccess will be exactly the same as the
+		// CountAfterAccess of the partial path (currently there's no index filter for partial path of mv index merge path).
```

Comment:
”currently there's no index filter for partial path of mv index merge path“
<img width="596" alt="image" src="https://github.com/pingcap/tidb/assets/15825830/bff4b32c-0b9a-4d85-9553-304d0c973b26">
we did?

Reply:
Which is this case? Is this a pure MV index path?

Reply:
o sorry, its not mv case